scripts/webhook-deploy.sh

```diff
@@ -54,6 +62,8 @@ done
 [ -z ${kubectl} ] && kubectl="kubectl"
 [ -z ${mode} ] && mode="preprogrammed"
 [ -z ${namespace} ] && namespace="default"
+[ -z ${uid} ] && uid="1000"
+[ -z ${gid} ] && gid="1000"
```

Comment:
I was thinking something like `$(id -u nobody)` (or some other suitable standard user that works)

Reply:
I'm sorry, I forgot to add creating fpga user. Now it should be done.

Reply:
done